ingenialink/virtual/network.py

```diff
@@ -0,0 +1,58 @@
+import socket
+
+import ingenialogger
+
+from ingenialink.constants import DEFAULT_ETH_CONNECTION_TIMEOUT
+from ingenialink.ethernet.network import NET_STATE, EthernetNetwork
+from ingenialink.exceptions import ILError
+from ingenialink.virtual.servo import VirtualServo
+
+logger = ingenialogger.get_logger(__name__)
+
+
+class VirtualNetwork(EthernetNetwork):
+    """Network for all virtual drive communications."""
+
+    def connect_to_slave(  # type: ignore [override]
+        self,
+        target: str,
+        dictionary: str,
+        port: int = 1061,
+        connection_timeout: float = DEFAULT_ETH_CONNECTION_TIMEOUT,
+        servo_status_listener: bool = False,
+        net_status_listener: bool = False,
+    ) -> VirtualServo:
+        """Connects to a slave through the given network settings.
+
+        Args:
+            dictionary: Path to the target dictionary file.
+            target: IP of the target slave.
```

Comment:
I think this value should be hardcoded, it will always be the loopback IP address.

Reply:
[4d83a54](https://github.com/ingeniamc/ingenialink-python/pull/361/commits/4d83a542b30cec1ae5dc384cfe1ca47743fa7c4f)